terraform/app_service.tf

```diff
@@ -45,7 +49,6 @@ resource "azurerm_linux_web_app" "main" {
   }
 
   app_settings = {
-    "DOCKER_REGISTRY_SERVER_URL"          = "https://ghcr.io/"
```

Comment:
initially i assumed that this (now invalid declaration) would need to be replaced with the block below.

```tf
site_config {
  application_stack {
    docker_registry_url = "https://ghcr.io/"
  }
}
```

but that's error inducing (unless you also specify `docker_image_name`_. to verify that the appropriate default is preserved, i ran a plan locally with this:

```tf
site_config {
  application_stack {
    python_version = "3.7"
  }
}
```
the plan output made me more confident that "https://ghcr.io" is still being set:

```tf
~ application_stack {
    - docker_image_name        = "cal-itp/benefits:c6af905c75df45d6d81a8f20fc3dc3bb2f472aae" -> null
    - docker_registry_url      = "https://ghcr.io/" -> null
    + python_version           = "3.7"
    # (10 unchanged attributes hidden)
}
```